source/use-cases/sensitive-data-encryption.txt

```diff
@@ -688,3 +730,21 @@ check out the reference docs in the server manual:
 - :manual:`Client-Side Field Level Encryption </core/security-client-side-encryption>`
 - :manual:`Automatic Encryption JSON Schema Syntax </reference/security-client-side-automatic-json-schema>`
 - :manual:`Manage Client-Side Encryption Data Keys </tutorial/manage-client-side-encryption-data-keys>`
+
+
+.. tabs-drivers::
+   :hidden: true
```

Comment:
nit: we generally prefer the "for x see y" construction, e.g.

"For additional information on CSFLE, see the official Java driver documentation"

Reply:
Fixed